python/ray/tests/test_autoscaler_e2e.py

```diff
@@ -14,6 +14,50 @@
 from ray.autoscaler.node_launch_exception import NodeLaunchException
 
 
+@pytest.mark.parametrize("enable_v2", [True, False])
+def test_ray_status_activity(shutdown_only, enable_v2):
+    reset_autoscaler_v2_enabled_cache()
+    cluster = AutoscalingCluster(
+        head_resources={"CPU": 0},
+        worker_node_types={
+            "type-i": {
+                "resources": {"CPU": 4, "fun": 1},
+                "node_config": {},
+                "min_workers": 1,
+                "max_workers": 1,
+            },
+            "type-ii": {
+                "resources": {"CPU": 3, "fun": 100},
+                "node_config": {},
+                "min_workers": 1,
+                "max_workers": 1,
+            },
+        },
+    )
+
+    try:
+        cluster.start(_system_config={"enable_autoscaler_v2": enable_v2})
+        ray.init(address="auto")
+
+        @ray.remote(num_cpus=2, resources={"fun": 2})
+        class Actor:
+            def ping(self):
+                return None
+
+        actor = Actor.remote()
+        ray.get(actor.ping.remote())
+
+        occurrences = 1 if enable_v2 else 0
+        assert (
+            subprocess.check_output("ray status --verbose", shell=True)
+            .decode()
+            .count("Resource: CPU currently in use.")
```

Comment:
Can we test that the `bundle_group_xxx` for placement groups resources are not showing up here?